Neos.Flow/Classes/ResourceManagement/PersistentResource.php

```diff
@@ -125,6 +133,14 @@ class PersistentResource implements ResourceMetaDataInterface, CacheAwareInterfa
      */
     protected $temporaryLocalCopyPathAndFilename;
 
+    /**
+     * @param string|resource|null $source
```

Comment:
```suggestion
     * The source is required for deferred persistence (after validation) in the framework and should otherwise not be set.
     *
     * @param string|resource|null $source
```